examples/custom-resources/backup-directive/transport-server/README.md

```diff
@@ -0,0 +1,111 @@
+# Support for Backup Directive in Transport Server
+
+NGINX Ingress Controller supports routing requests to a service specified as `Backup`.
+The `backup` service is of type
+[ExternalName](https://kubernetes.io/docs/concepts/services-networking/service/#externalname).
+
+> [!NOTE]
+> Support of the services of type
+> [ExternalName](https://kubernetes.io/docs/concepts/services-networking/service/#externalname)
+> is only available in NGINX Plus.
```

Comment:
```suggestion
> The [ExternalName](https://kubernetes.io/docs/concepts/services-networking/service/#externalname) service is only available with NGINX Plus.
```

This phrasing is more concise, and brings the subject (The ExternalName service) to the start of the sentence.

I suggest "with" instead of "in", because "In" suggests that it's NGINX Plus managing the service, not NGINX Ingress Controller.